### ISSUE TITLE
missing tab order for encrypted reply

### ISSUE THREAD

**Issue by tomholub** [FlowCrypt/flowcrypt-browser#2094]
![image](https://user-images.githubusercontent.com/6306961/66402523-7cf0b880-e9d4-11e9-9163-dba735fd7a3b.png)

The green one. Cannot access it with a tab to reply
